compiler/src/codegen/mashtree.re

```diff
@@ -429,7 +429,7 @@ and block = list(instr);
 [@deriving sexp]
 type import_type =
   | MFuncImport(list(asmtype), list(asmtype))
-  | MGlobalImport(asmtype);
+  | MGlobalImport(asmtype, bool);
```

Comment:
In general, I suppose so, yeah. This one sorta makes sense because globals went from always being immutable to optionally being mutable. I think in general we could prefer the record-type variants since those are self-documenting.